scripts/nginx_conf_part_2.conf

```diff
@@ -48,14 +48,14 @@ server {
     index index.html index.htm;   
 
     location / {
-        try_files $uri $uri/ /index.html;
```

Comment:
i think we still want `/index.html` in that chain since sometimes the URLs will be like 

`www.example.com/something/`

 which we'll want to load 

`www.example.com/something/index.html`